Xamarin.Forms.Core.UnitTests/MultiBindingTests.cs

```diff
@@ -33,16 +33,8 @@ public override void TearDown()
 		}
 
 		[Test]
-		public void TestChildOneWayOverridesMultiTwoWay()
+		public void TestChildOneWayOnMultiTwoWay()
 		{
-			// This tests a weird edge case where the MultiBinding is TwoWay but
-			// one of the child bindings is OneWay. This results in a situation where
-			// you actually can't change the target value to be inconsistent with the
-			// the source value for the OneWay because ConvertBack causes Convert
-			// to get re-evaluated (and this effectively negates the change to the
-			// target). This ensures that the target and source never go out of sync,
-			// as the overall binding is still TwoWay.
```

Comment:
changed Multibinding to use target BP default mode in case of none specified. sub Bindings defaults to the same mode.

it's totally fine to have an intermediate model out of sync, it's better than forcing 1W bindings to behave 2W

Reply:
Making the default mode match that of the BP sounds good but as for the second part of that, I was trying to make sure that if a multibinding is two-way but a child is one-way, the target's value can't become inconsistent with any of the sources of the one-way bindings. That feels like the right behavior and is consistent with WPF; not sure if your changes actually impacted that or not.